torchgeo/datasets/eudem.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""European Digital Elevation Model (EU-DEM) dataset."""
+
+import glob
+import os
+from typing import Any, Callable, Dict, Optional
+
+import matplotlib.pyplot as plt
+from rasterio.crs import CRS
+from torch import Tensor
+
+from .geo import RasterDataset
+from .utils import check_integrity, extract_archive
+
+
+class EUDEM(RasterDataset):
+    """EUDEM Dataset.
+
+    The `EUDEM
```

Comment:
```suggestion
    """EU-DEM Dataset.

    The `EU-DEM
```
Or copy the longer name from the file docstring.